src/inmanta/data/dataview.py

```diff
@@ -98,7 +98,28 @@
 
 
 class RequestedPagingBoundaries:
-    """Represents the lower and upper bounds that the user requested for the paging boundaries, if any."""
+    """
+    Represents the lower and upper bounds that the user requested for the paging boundaries, if any.
+
+    Boundary values represent min and max values, regardless of sorting direction (ASC or DESC), i.e.
+                  |        prev             |                current page                    |    next
+     -------------|-------------------------|------------------------------------------------|----------------------
+     ASC sorting  | [  ...  ] (99 c)        | [ (100 d) ( 100 e) ... (10 000 r) (10 000 s) ] | (10 001 t) [ ...   ]
+                  |            end = 99     |     start = 100          end = 10 000          |     start = 10 001
+                  |            last_id = c  |     first_id = d         last_id = s           |     first_id = t
+                  |
+     DESC sorting | [  ...  ] (10 001 t)    | [ (10 000 s) (10 000 r) ... (100 d) ( 100 c) ] |  (99 b) [ ...   ]
+                  |          start = 10 001 |         end = 10000          start = 100       |     end = 99
+                  |          first_id = t   |         last_id = s          first_id = c      |     last_id = b
```

Comment:
Same here, I'm not so sure of what we're conveying here, maybe adds more confusion than anything